### PR TITLE
Fix broken link to GWT Eclipse Plugin Logo

### PR DESCRIPTION
**Pull request by TDesjardins**
This change is [Reviewable](https://reviewable.io/reviews/gwtproject/gwt-site/274)

### CODE REVIEW

File: src/main/markdown/download.md [hunk 1]
```diff
@@ -59,7 +59,7 @@ Download
   </div>
 </div>
 
-<img class='download-icon' src="http://gwt-plugins.github.io/documentation/gwt-eclipse-plugin/workspace/images/logo.png" />
+<img class='download-icon' src="http://gwt-plugins.github.io/documentation/images/logo.png" />
```
Comment (tbroyer) [inline src/main/markdown/download.md]: Should we take that opportunity to use https too?

**Comment by TDesjardins** [comment]
I started a request to Eclipse Foundation but if you ask me I would simply remove the icon and then also consistently remove the old-fashioned icon for the GWT-SDK.

Looks then:
gwt-download

If you agree I will fit the PR.

**Comment by TDesjardins** [comment]
Done!

**Comment by tbroyer** [comment]
Thanks

**Pull request merged**